roles/nvidiadl_ssd_benchmark_deploy/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Create the entrypoint ConfigMap
+  command: 
+    oc create cm custom-config-script --from-file="{{ entrypoint }}"
```

Comment:
could you make sure that the benchmark can be executed twice in a row? 
`oc create` fails if the resource already exists, `oc apply` doesn't :)